roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -33,6 +37,7 @@
     group: root
   with_items:
     - "{{ ceph_keys.stdout_lines }}"
+  run_once: true
```

Comment:
Again I might be wrong but we may need to set the file permissions on all monitor hosts, right?